rapid7/metasploit-framework
# Add support for rails 7

Upgrade to Rails 7.0.

Brings the framework environment up to the current version of Rails.

## Verification

* Start `msfconsole` with various database states
- [ ] no database
- [ ] database configured via `msfdb init`
- [ ] database config in `~/.msf4/database.yml` based on `config/database.yml.example`
- [ ] database config in `config/database.yml` based on `config/database.yml.example`

* **Verify** database commands function as expected
- [ ] `hosts`
- [ ] `vulns`
- [ ] `db_import`
- [ ] `db_nmap`

* **Verify** modules function as expected
- [ ] tester selected module 1
- [ ] tester selected module 2
- [ ] tester selected module 3
- [ ] tester selected module 4

* **Verify** sessions interactions
- [ ] windows meterpreter session (created with psexec)
- [ ] windows shell session (created with psexec)
- [ ] linux meterpreter session (created with `msfvenom`)


Depends on (updates to PR will be made as gems release):
https://github.com/rapid7/metasploit-concern/pull/40
https://github.com/rapid7/metasploit_data_models/pull/200
https://github.com/rapid7/metasploit-credential/pull/167
